app.py

```diff
@@ -1,12 +1,29 @@
-from flask import Flask
+from flask import Flask, jsonify
+import requests
 
 app = Flask(__name__)
 
 
+@app.errorhandler(requests.exceptions.RequestException)
+def handle_request_exception(e):
+    return jsonify({"error": "could not make request, please see an admin"}), 500
+
 @app.route('/')
 def hello_world():
     return 'Hello World!'
 
 
+@app.route('/local-hospital-capacity', methods=['GET'])
+def get_cap():
+    responses = requests.get('https://covid19-server.chrismichael.now.sh/api/v1/AggregatedFacilityCapacityCounty').json()['data'][0]['table']
+    tx_response = []
+    for response in responses:
+        if response['State'] == 'TX' and response['CountyName'] == 'Bexar':
```

Comment:
Are we planning on changing this later to not be hard coded to Bexar and TX?

Reply:
yeah i think it should be pretty easy to add in some querying bits in the url, its not much more code than whats already been written